# COVID_RNAseq_script

Total RNA sequencing 
RNA was ribodepleted, and globin depleted using Ribo-ZeroTM Gold rRNA removal Kit (Illumina, USA). RNA was converted to cDNA; second-strand cDNA synthesis incorporated dUTP. The cDNA was end-repaired, A-tailed and adapter-ligated, and prior to amplification, samples underwent uridine digestion. The prepared libraries were size-selected, multiplexed, and quality controlled before 150bp paired-end sequencing (NovaSeq6000). Sequencing was conducted at the Wellcome Trust Centre for Human Genetics (Oxford, UK). Sequences that mapped to human rRNA genome were excluded and the remaining sequencing data were aligned against the whole human (Homo sapiens) genome build GRCh38 (https://ccb.jhu.edu/software/hisat2/index.shtml), using STAR (version 2.7.3a). Gene features were counted using HTSeq (version 0.11.1), using human gene annotation general transfer format version GRCh38.92 (www.ensembl.org).Gene features were counted using HTSeq (version 0.11.1), using human gene annotation general transfer format version GRCh38.92 (www.ensembl.org).Gene features were counted using HTSeq (version 0.11.1), using human gene annotation general transfer format version GRCh38.92 (www.ensembl.org).  
